docs/CodeStyle.md

```diff
@@ -32,6 +32,13 @@
 ## C++
   * used only for unit testing
   * lower-case class names (same as stl/boost)
+  * the tests are written using [C++11](https://en.cppreference.com/w/cpp/11). Whenever applicable, the usage of advanced language features is allowed and preferred over legacy code. For example:
```

Comment:
"the tests" - "the unit tests in test/gtest"

Reply:
Fixed